src/components/profile/profile.tsx

```diff
@@ -49,6 +51,17 @@ class Profile extends React.Component<Props> {
         if (!this.props.currentState) {
             this.props.getStateByPublicKey(this.props.currentUser.publicKey);
         }
+
+        // @ts-ignore
+        this.balanceRefreshTimerId = setInterval(() => {
+            if (this.props.currentUser && this.props.currentUser.publicKey) {
+                this.props.getStateByPublicKey(this.props.currentUser.publicKey);
```

Comment:
It might be my lack of knowledge on the application here but wouldn't we want to update the Redux state with some sort of set function rather than only calling a getter function? Will this ever update the Redux state?

Reply:
This uses the same format as `this.props.getTransactionsByRecipient(this.props.currentUser.publicKey)` made by @roelvdboom. If I understand correctly the Redux state is updated in state.reducer.ts when a state handler is triggered in state.actions.ts by the `getStateByPublicKey` API call.

Reply:
@saltz What do you think about this? ^^ 

Personally, I wouldn't expect a getter function to change any values.

Reply:
@TimoF97 If you look at the whole application. This is the naming convention we used. If you want to change this we should do this in another pull request.